stuffer/s2n_stuffer_file.c

```diff
@@ -28,40 +28,47 @@
 
 int s2n_stuffer_recv_from_fd(struct s2n_stuffer *stuffer, int rfd, uint32_t len)
```

Comment:
All these should really have a `uint32_t* bytes_written` parameter, and return `s2n_error`

Reply:
Done!